packages/components/src/components/primitives/PinInput/PinInput.tsx

```diff
@@ -0,0 +1,7 @@
+import React, { FC } from 'react';
+import { PinInput as ChakraPinInput, PinInputField as ChakraPinInputField } from '@chakra-ui/react';
+import { PinInputProps, PinInputFieldProps } from './types';
+
+export const PinInput: FC<PinInputProps> = (props) => <ChakraPinInput {...props} data-testid="cmpsr.pin-input" />;
+
+export const PinInputField: FC<PinInputFieldProps> = (props) => <ChakraPinInputField {...props} />;
```

Comment:
Could we change the implementation so `PinInput` accepts a new property, something like `numOfDigits` and given that value we render the `PinInputField` ?

Reply:
Yes, I think it makes a lot of sense. I am going to create a new ticket to work on that. What do you think about passing props to `PinInputField` let's discuss that within the [ticket](https://impulsumvc.atlassian.net/browse/COM-424).